src/main/java/io/vertx/redis/client/impl/RedisURI.java

```diff
@@ -27,14 +27,42 @@
 import java.util.Map;
 
 /**
- * Utility to parse redis URLs. The URI should follow the scheme: redis://[username:password@][host][:port][/[database]
- * An example URI can be found at <a href="https://redis.io/topics/rediscli">Redis cli docs</a>
+ * Utility to parse Redis URLs. The URI should follow the general scheme:
+ * {@code redis://[username:password@][host][:port][/[database]}
```

Comment:
The change seems reasonable, but then this pattern needs to be adjusted. I believe it should look like this now:

```suggestion
 * {@code redis://[[username][:password]@][host][:port][/[database]]}
```